src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { funFacts, FunFact } from "./funfacts"
+import { useState } from "react"
+import "./FunFactPage.scss"
+
+interface FunFactsProps {
```

Comment:
This should match the component name, so (taking the below comment into account):
```suggestion
interface FunFactProps {
```

Reply:
Done, thanks